src/objects.c

```diff
@@ -1179,6 +1182,8 @@ static CK_RV get_all_attrs(P11PROV_OBJ *obj, CK_ATTRIBUTE *attrs, int num)
     CK_ATTRIBUTE *res[num];
     CK_RV rv;
 
+    P11PROV_debug("called (obj=%p, atrs=%p, num=%d)", obj, attrs, num);
+
```

Comment:
This is redundant as get_public_attrs never modifies the data, only selects where to get it from.